nais.yaml

```diff
@@ -1,33 +1,35 @@
-name: pam-er-sync
-team: teampam
-image: repo.adeo.no:5443/pam-er-sync
-replicas:
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: pam-er-sync
+  namespace: default
+  labels:
+    team: teampam
+spec:
+  image: repo.adeo.no:5443/pam-er-sync
+  port: 9012
+  replicas:
     min: 1
     max: 1
     cpuThresholdPercentage: 50
-prometheus:
+  prometheus:
     enabled: true
     path: /actuator/prometheus
-port: 9012
-healthcheck:
-    liveness:
-      initialDelay: 120
-      path: /actuator/info
-    readiness:
-      initialDelay: 120
-      path: /actuator/health
-resources:
+  liveness:
+    initialDelay: 20
```

Comment:
Nice! Har du testa at appen faktisk starter på under 20 sekunder?

Reply:
Localt så tar hele greia 50s, men det inkl bygg også.
Fra loggen ser jeg at spring  boot run tar under 20s

Reply:
Kan evt. øke den for å være på sikre siden